phpBB/styles/prosilver/template/simple_footer.html

```diff
@@ -1,8 +1,19 @@
 	</div>
 
-	<div class="copyright" role="contentinfo">{CREDIT_LINE}
-		<!-- IF TRANSLATION_INFO --><br />{TRANSLATION_INFO}<!-- ENDIF -->
-		<!-- IF DEBUG_OUTPUT --><br />{DEBUG_OUTPUT}<!-- ENDIF -->
+	<div class="copyright" role="contentinfo">
+		<p class="footer-row">
+			<span class="footer-copyright">{{ CREDIT_LINE }}</span>
+		</p>
+		{% if TRANSLATION_INFO %}
+			<p class="footer-row">
+				<span class="footer-copyright">{{ TRANSLATION_INFO }}</span>
```

Comment:
Same class as the Credit_line in Line 5? Maybe a "footer-translation-info" would be more sufficient.

Reply:
This class is identical to the class used in the `overall_footer.html`: 
https://github.com/phpbb/phpbb/blob/master/phpBB/styles/prosilver/template/overall_footer.html#L16
So I would in favor of keeping the same class as there.

Reply:
Yeah, then better fix both later in a new PR.